controls/V-72081.rb

```diff
@@ -1,96 +1,107 @@
 # encoding: utf-8
 #
-=begin
------------------
-Benchmark: Red Hat Enterprise Linux 7 Security Technical Implementation Guide
-Status: Accepted
-
-This Security Technical Implementation Guide is published as a tool to improve
-the security of Department of Defense (DoD) information systems. The
-requirements are derived from the National Institute of Standards and
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
-
-Release Date: 2017-03-08
-Version: 1
-Publisher: DISA
-Source: STIG.DOD.MIL
-uri: http://iase.disa.mil
------------------
-=end
+
+MONITOR_KERNEL_LOG = attribute(
```

Comment:
Please change global variables to local so that inspec does not run into issues.

Reply:
@samcornwell Tested this with the global removed and happy to patch but you're not allowing pushes from maintainers. If you just downcase this in the file, we can get this through.

Reply:
@aaronlippold Can you get this fix in so that we can push this? Allow updates from maintainers doesn't appear to be checked so I can't fix it from my end.